flytekit/models/sagemaker/parameter_ranges.py

```diff
@@ -177,8 +178,15 @@ def to_flyte_idl(self) -> _idl_parameter_ranges.ParameterRanges:
                 converted[k] = _idl_parameter_ranges.ParameterRangeOneOf(integer_parameter_range=v.to_flyte_idl())
             elif isinstance(v, ContinuousParameterRange):
                 converted[k] = _idl_parameter_ranges.ParameterRangeOneOf(continuous_parameter_range=v.to_flyte_idl())
-            else:
+            elif isinstance(v, CategoricalParameterRange):
```

Comment:
Can you pull Master? This check actually doesn't work as expected. I have merged my fix.

Reply:
What are you referring to exactly? [this](https://github.com/lyft/flytekit/blob/master/flytekit/models/sagemaker/parameter_ranges.py#L169-L183) is the copy on master

Reply:
My bad. I thought this was `from_flyte_idl()`

Reply:
So, all good?